app/Http/Controllers/AdminConabController.php

```diff
@@ -0,0 +1,105 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\User;
+use Illuminate\Http\Request;
+use Illuminate\Support\Facades\Auth;
+use Illuminate\Support\Facades\Validator;
+use Illuminate\Support\Facades\Hash;
+
+class AdminConabController extends Controller
+{
+    /**
+     * Display a listing of the resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function index()
+    {
+        $user = Auth::user();
+        $users = User::where('user_type', 'ADMIN_CONAB')->where('id', '<>', $user->id)->get();
+        return $users;
+    }
+
+    /**
+     * Store a newly created resource in storage.
+     *
+     * @param  \Illuminate\Http\Request  $request
+     * @return \Illuminate\Http\Response
+     */
+    public function store(Request $request)
+    {
+        $data = Validator::make($request->all(), [
+            'name' => 'required|string',
+            'email' => 'required|email',
+            'cpf' => 'required|regex:/^[0-9]{3}\.[0-9]{3}\.[0-9]{3}\-[0-9]{2}/',
+            'phones' => 'required',
+            'phones.*.number' => 'required|string|regex:/^\([0-9]{2}\) [0-9]{5}\-[0-9]{4}/'
+        ])->validate();
+
+        $user = new User();
+        $user->fill($data);
+        $user->password = $data['cpf'];
+        $user->user_type = 'ADMIN_CONAB';
+        $user->save();
+        $phones = $user->phones()->createMany($data['phones']);
+        $userAndPhones = array_merge($user->toArray(), [ 'phones' => $phones ]);
+
+        return response($userAndPhones, 201);
+    }
+
+    /**
+     * Update the specified resource in storage.
+     *
+     * @param  \Illuminate\Http\Request  $request
+     * @param  int  $id
+     * @return \Illuminate\Http\Response
+     */
+    public function update(Request $request, $id)
+    {
+        $admin = User::with('phones')->findOrFail($id);
+
+        $data = Validator::make($request->all(), [
+            'name' => 'string',
+            'email' => 'string|email',
+            'cpf' => 'regex:/^[0-9]{3}\.[0-9]{3}\.[0-9]{3}\-[0-9]{2}/',
```

Comment:
Tanto email como o CPF devem ser únicos.

Reply:
@lucasemanuel8 não vai dá pra fazer isso no update, pq quando a gente for mandar os dados no frontend nós não temos como saber quais os dados que o usuário alterou, por isso nós enviamos todos os dados, ae o método update do laravel ver quais mudaram e faz a alteração. Então pense comigo, se eu mandar todos os dados, porém só com o nome alterado, os outros dados já existiram no banco. Então a validação nunca passaria para o cpf e/ou os phones, pq o usuário que está alterando o nome já possui o cpf/phone cadastrado.

Reply:
Você pode ignorar os dados persistidos no model da seguinte maneira.
```
Rule::unique('admin')->ignore($admin->id)
```